src/rabbit_policies.erl

```diff
@@ -104,7 +105,13 @@ validate_policy0(<<"queue-mode">>, <<"default">>) ->
 validate_policy0(<<"queue-mode">>, <<"lazy">>) ->
     ok;
 validate_policy0(<<"queue-mode">>, Value) ->
-    {error, "~p is not a valid queue-mode value", [Value]}.
+    {error, "~p is not a valid queue-mode value", [Value]};
+validate_policy0(<<"overflow">>, <<"drop_head">>) ->
```

Comment:
The other policy names use dash characters instead of underscores. Should these be `drop-head` and `reject-publish` to be consistent? They could still be used as atoms elsewhere if single-quoted.

Reply:
Good find. Will change.